internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
```

Comment:
I don't think so, no. We aren't reporting if it the connection is up or down like with Test-Connection or even Test-SqlConnection. We are simply reporting back lookups.